package com.byazt.rl;

import client.android.yixiaotong.v3.comman.AppTypeIdUtil;
import com.bykv.vk.component.ttvideo.player.MediaPlayer;
import com.bykv.vk.openvk.api.proto.Result;
import com.bykv.vk.openvk.api.proto.ValueSet;

/* JADX INFO: loaded from: C:\Users\MR\AppData\Local\Temp\yixiaotong-dex\5941276.dex */
@com.byazt.zqa.c(c = {0, 1, AppTypeIdUtil.Drink4GCostControl_1, 13})
public class tt {
    public boolean c = false;
    public int tt = -1;
    public String ve = null;
    public ValueSet uj = null;

    /* JADX INFO: renamed from: com.byazt.rl.tt$1, reason: invalid class name */
    @com.byazt.zqa.c(c = {0, 1, AppTypeIdUtil.Drink4GCostControl_1, 28})
    static /* synthetic */ class AnonymousClass1 {
    }

    public tt c(int i) {
        this.tt = i;
        return this;
    }

    public tt c(ValueSet valueSet) {
        this.uj = valueSet;
        return this;
    }

    public tt c(String str) {
        this.ve = str;
        return this;
    }

    public tt c(boolean z) {
        this.c = z;
        return this;
    }

    private tt() {
    }

    public static final tt c() {
        return new tt();
    }

    public Result tt() {
        boolean z = this.c;
        int i = this.tt;
        String str = this.ve;
        ValueSet valueSetTt = this.uj;
        if (valueSetTt == null) {
            valueSetTt = ve.c().tt();
        }
        return new c(z, i, str, valueSetTt, null);
    }

    @com.byazt.zqa.c(c = {0, 1, AppTypeIdUtil.Drink4GCostControl_1, MediaPlayer.MEDIA_PLAYER_OPTION_JX_CODEC_LOW_LATENCY})
    private static final class c implements Result {
        public final boolean c;
        public final int tt;
        public final ValueSet uj;
        public final String ve;

        @Override // com.bykv.vk.openvk.api.proto.Result
        public int code() {
            return this.tt;
        }

        @Override // com.bykv.vk.openvk.api.proto.Result
        public boolean isSuccess() {
            return this.c;
        }

        @Override // com.bykv.vk.openvk.api.proto.Result
        public String message() {
            return this.ve;
        }

        @Override // com.bykv.vk.openvk.api.proto.Result
        public ValueSet values() {
            return this.uj;
        }

        public /* synthetic */ c(boolean z, int i, String str, ValueSet valueSet, AnonymousClass1 anonymousClass1) {
            this(z, i, str, valueSet);
        }

        private c(boolean z, int i, String str, ValueSet valueSet) {
            this.c = z;
            this.tt = i;
            this.ve = str;
            this.uj = valueSet;
        }
    }
}
